fe/fe-core/src/main/java/org/apache/doris/planner/external/paimon/PaimonScanNode.java

```diff
@@ -129,7 +138,28 @@ public List<Split> getSplits() throws UserException {
                 .newScan().plan().splits();
         for (org.apache.paimon.table.source.Split split : paimonSplits) {
             PaimonSplit paimonSplit = new PaimonSplit(split);
-            splits.add(paimonSplit);
+            if (split instanceof DataSplit) {
+                DataSplit dataSplit = (DataSplit) split;
+                Optional<List<RawFile>> optRowFiles = dataSplit.convertToRawFiles();
+                if (optRowFiles.isPresent()) {
+                    List<RawFile> rawFiles = optRowFiles.get();
+                    rawFiles.forEach(file -> {
```

Comment:
We need to take care of the file split, in case there are large file to be read in one thread.
You can see the `splitFile()` method in `FileScanNode`.

For example, in hive scan node, we will get the origin file splits first, than split these files again
by size(default is 128MB)